### PR TITLE
Update homebrew support

### PR DESCRIPTION
**Pull request by tim-coutinho**
This addresses #218.

### CODE REVIEW

File: shallow_backup/backup.py
Comment (alichtman) [inline shallow_backup/backup.py:155]:
```suggestion
			print_yellow("brew package manager not found.")
```

File: shallow_backup/backup.py
Comment (alichtman) [inline shallow_backup/backup.py:141]: This should be refactored. `for x in [item]` is an anti-pattern.

**Review by alichtman — CHANGES_REQUESTED** [review]
Quick refactor and then all good.

**Comment by alichtman** [comment]
Note that this should be merged after #292 for minimal merge conflicts.

File: shallow_backup/backup.py
Comment (tim-coutinho) [reply shallow_backup/backup.py]: True. I was paring this down from the larger PR and it left it like this. This'll be changed with the ruby PR regardless.

File: shallow_backup/backup.py
Comment (alichtman) [inline shallow_backup/backup.py:153]:
```suggestion
		if not run_cmd(command):
```

**Comment by alichtman** [comment]
Good to go after this change.

**Comment by finn-matti** [comment]
Is this still ongoing? And if not, what needs to be done? I am not a python dev, but I would love for this to fix the brew integration.

**Pull request merged**
